gadget/layout.go

```diff
@@ -63,8 +63,6 @@ type LaidOutVolume struct {
 	// LaidOutStructure is a list of structures within the volume, sorted
 	// by their start offsets
 	LaidOutStructure []LaidOutStructure
-	// RootDir is the root directory for volume data
-	RootDir string
```

Comment:
so this wasn't used?

Reply:
It wasn't... must have been a leftover :man_shrugging: